Core/NoOpRunner.Core/RandomNumber.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NoOpRunner.Core
+{
+    class RandomNumber
```

Comment:
[**Q**] Why do we need this?

Reply:
To randomly generate numbers. Otherwise all levels of platforms and power-up spawn locations would be near identical.